src/LibraryManager.Vsix/UI/Controls/TargetLocation.xaml.cs

```diff
@@ -279,5 +286,26 @@ private void TargetLocation_LostFocus(object sender, RoutedEventArgs e)
                 Flyout.IsOpen = false;
             }
         }
+
+        private void LibraryNameChanged(object sender, PropertyChangedEventArgs e)
+        {
+            string targetLibrary = MutualPropertyChange.Instance.TargetLibrary;
+
+            this.Dispatcher.Invoke(() =>
+            {
+                if (TargetLocationSearchTextBox.Text.Equals(_lastTargetLocation))
+                {
+                    if (targetLibrary.Length > 0 && targetLibrary[targetLibrary.Length - 1] == '/')
+                    {
+                        targetLibrary = targetLibrary.Substring(0, targetLibrary.Length - 1);
+                    }
```

Comment:
Prafull: The provider will have a suggestion for what folder should be appended. We want to match what the CLI would suggest, so look at that logic.

Reply:
Yes. I am able to use SelectedPackage to get the suggested destination for the selectedProvider

---
In reply to: [200772301](https://github.com/aspnet/LibraryManager/pull/285#discussion_r200772301) [](ancestors = 200772301)